package.json

```diff
@@ -3,15 +3,16 @@
   "version": "1.0.0",
   "description": "Gladia documentation",
   "main": "index.js",
+  "author": "",
```

Comment:
Should we specify the author as being one of our shared email address or just "Gladia Tech team"?